bindings/python/iota_sdk/wallet/account.py

```diff
@@ -343,12 +343,12 @@ def prepare_output(self, params: OutputParams,
            When the assets contain an nft_id, the data from the existing nft output will be used, just with the address
            unlock conditions replaced
         """
-        return from_dict(Output, self._call_account_method(
+        return output_from_dict(self._call_account_method(
```

Comment:
Changelog that this fixes prepare_output()

Reply:
done